themes/bootstrap5/templates/comments/usercomments.phtml

```diff
@@ -0,0 +1,94 @@
+      <?php if ($this->accountCapabilities()->isRatingRemovalAllowed() || ('enabled' === $this->accountCapabilities()->getCommentSetting())): ?>
+        <input type="hidden" value="<?=$this->escapeHtmlAttr($this->auth()->getManager()->getCsrfHash())?>" name="csrf">
+        <label>
+          <div class="checkbox">
+            <input type="checkbox" name="selectAll" class="checkbox-select-all">
+            <?=$this->transEsc('select_all_on_page')?>
+          </div>
+        </label>
+        <?=
+          $this->component(
+              'confirm-button',
+              [
+                  'buttonId' => 'cancelSelected',
+                  'buttonName' => 'cancelSelected',
+                  'buttonLabel' => 'delete_selected',
+                  'header' => 'confirm_usercomment_delete_selected_text',
+                  'confirmId' => 'confirm_cancel_selected_yes',
+                  'cancelClass' => 'confirm_cancel_no',
+                  'ignoreLightbox' => true,
+              ]
+          )
+        ?>
+      <?php endif; ?>
+      <table class="usercomments-table table-striped table table-responsive">
+        <thead>
+          <tr>
+            <th></th>
+            <th><?=$this->transEsc('Created')?></th>
+            <th><?=$this->transEsc('Title')?></th>
+            <th><?=$this->transEsc('Comment or Rating')?></th>
+          </tr>
+        </thead>
+        <tbody>
+          <?php foreach ($this->comments as $comment):?>
+            <tr>
+              <td class="checkbox-col">
+                <?php $className = (null === $comment['comment']) ? 'rating' : 'comment'; ?>
+                <?php if ($className === 'comment' || ($className === 'rating' && $this->accountCapabilities()->isRatingRemovalAllowed())):?>
+                  <div class="checkbox-container result">
+                    <div class="checkbox">
+                      <label>
+                        <span class="sr-only"><?=$this->transEsc('Select');?></span>
+                        <?php $className = (null === $comment['comment']) ? 'rating' : 'comment';?>
+                        <input class="checkbox-select-item" type="checkbox" name="deleteSelected<?=$className?>[]" value="<?=$this->escapeHtmlAttr(($comment[$className . '_id'] ?? ''))?>" id="checkbox_<?=$className?>_<?=$comment[$className . '_id']?>">
+                      </label>
+                    </div>
+                  </div>
+                <?php endif; ?>
+              </td>
+              <td><?=$this->dateTime()->convertToDisplayDateAndTime('Y-m-d H:i:s', $comment['created'])?></td>
+              <td>
+                <a href="<?=$this->url('record', ['id' => $comment['record_id']])?>"><?=$comment['recordTitle']?></a>
+              </td>
+              <td class="user-comment">
+                <?php if (isset($comment['rating'])): ?>
+                  <?=$this->render('Helpers/star-rating.phtml', ['readonly' => true, 'ratingData' => ['rating' => $comment['rating'], 'count' => 1]])?>
```

Comment:
This will require minor adjustment if/when #4317 is merged -- just something to keep an eye on depending on which part gets done first.